BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -683,12 +683,17 @@ def adjust_auth_type (auth_type_str, key_path):
     return auth_type_str
 
 def gen_layout (comp_list, img_type, auth_type_str, svn, out_file, key_dir, key_file):
+    if auth_type_str in ['SHA2_256','SHA2_384']:
+        raise Exception ("Invalid 'auth' parameter '%s'. Monolithic signing requires a signature-type auth." % (auth_type_str))
```

Comment:
I think we should allow only hash auth also,
This is common function for any type of container

Reply:
This wasn't working previously for monolithic signing, I just added the more clear explanation.